osu.Game.Rulesets.Mania/UI/DrawableManiaRuleset.cs

```diff
@@ -76,7 +85,8 @@ private void load()
             Config.BindWith(ManiaRulesetSetting.ScrollDirection, configDirection);
             configDirection.BindValueChanged(direction => Direction.Value = (ScrollingDirection)direction.NewValue, true);
 
-            Config.BindWith(ManiaRulesetSetting.ScrollTime, TimeRange);
+            Config.BindWith(ManiaRulesetSetting.ScrollTime, configTimeRange);
+            configTimeRange.BindValueChanged(_ => updateTimeRange());
```

Comment:
Gave me a little pause - maybe a bit gratuitous if `updateTimeRange` is being called inside `Update()` anyways...? It won't have any effect as the underlying `BindableDouble` won't fire `ValueChanged` if the value is the same twice in one frame but still reads a little weird.

Reply:
Sure. Only did it for consistency with the above, so I'm fine with removing it.